src/app/common/data.helpers.ts

```diff
@@ -175,6 +175,34 @@ export class DataHelpers {
         }
     }
 
+    public static descriptionForDatasetFlow(flow: FlowSummaryDataFragment): string {
+        const decriptionFlow = flow.description;
+        switch (decriptionFlow.__typename) {
+            case "FlowDescriptionDatasetPollingIngest": {
+                return `Polling ingest`;
+            }
+            case "FlowDescriptionDatasetPushIngest": {
+                {
+                    return `Push ingest`;
+                }
+            }
+            case "FlowDescriptionDatasetExecuteTransform": {
+                {
+                    return `Execute transformation`;
+                }
+            }
+            case "FlowDescriptionDatasetCompaction": {
+                return `Execute compaction`;
+            }
+            case "FlowDescriptionSystemGC": {
+                return `SystemGC work`;
+            }
```

Comment:
This will never appear in dataset flows.

Reply:
Fixed